src/code/z_player_lib.c

```diff
@@ -374,7 +443,7 @@ void Player_SetModelGroup(Player* this, s32 modelGroup) {
     Player_SetModels(this, modelGroup);
 }
 
-void func_8008EC70(Player* this) {
+void Player_UnsetItemAction(Player* this) {
     this->itemActionParam = this->heldItemActionParam;
     Player_SetModelGroup(this, Player_ActionToModelGroup(this, this->heldItemActionParam));
     this->unk_6AD = 0;
```

Comment:
This function doesnt really have to do with unsetting anything.
Basic testing and a little bit of reading shows its related specifically to put an old item back in links hand after using a custscene item (ocarina, magic spell, etc). But there are probably more contexts where its used so I wouldnt feel comfortable naming this yet

I am planning a docs pass of the item system/action parameters specifically, so you can leave this unnamed and Ill get to it later with a deeper understanding of the system